build/Versions.props

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the MIT license. See License.txt in the project root for full license information. -->
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+  </PropertyGroup>
+
+  <!-- Repo Version Information -->
+  <PropertyGroup>
+    <VersionBase>2.0.3</VersionBase>
+    <PreReleaseVersionLabel>servicing</PreReleaseVersionLabel>
+  </PropertyGroup>
+
+  <!-- Repo Toolset Features -->
+  <PropertyGroup>
+    <UsingToolMicrosoftNetCompilers>false</UsingToolMicrosoftNetCompilers>
+    <UsingToolSourceLink>false</UsingToolSourceLink>
+    <UsingToolVSSDK>true</UsingToolVSSDK>
+  </PropertyGroup>
+
+  <!-- Toolset Dependencies -->
+  <PropertyGroup>
+    <DotNetCliVersion>2.0.2-vspre-006963</DotNetCliVersion>
+    <MicrosoftDotNetBuildTasksFeedVersion>1.0.0-prerelease-01929-02</MicrosoftDotNetBuildTasksFeedVersion>
+    <RoslynToolsMicrosoftRepoToolsetVersion>1.0.0-alpha52</RoslynToolsMicrosoftRepoToolsetVersion>
+    <RoslynToolsMicrosoftSignToolVersion>0.3.7-beta</RoslynToolsMicrosoftSignToolVersion>
```

Comment:
><RoslynToolsMicrosoftSignToolVersion>0.3.7-beta</RoslynToolsMicrosoftSignToolVersion> [](start = 4, length = 85)

Not necessary. The toolset itself references version that it needs to do sign.

Reply:
This is a newer version than what repo-toolset is consuming by default.

Reply:
Feel free to update the toolset to this version.